stack_master.gemspec

```diff
@@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "table_print"
   spec.add_dependency "dotgpg"
   spec.add_dependency "deep_merge"
+  spec.add_dependency "cfndsl"
```

Comment:
Yep. That too. Noted.

On Wed, 6 Apr 2016 at 14:14 Steve Hodgkiss <notifications@github.com> wrote:

> In stack_master.gemspec
> <https://github.com/envato/stack_master/pull/99#discussion_r58649082>:
>
> > @@ -36,4 +36,5 @@ Gem::Specification.new do |spec|
> >    spec.add_dependency "table_print"
> >    spec.add_dependency "dotgpg"
> >    spec.add_dependency "deep_merge"
> > +  spec.add_dependency "cfndsl"
>
> Cool. We'll also need to update the readme about the fact that they need
> to be installed separately...
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/envato/stack_master/pull/99/files/70581ad71e83414cef73bf2c2f14df51d99e9c69#r58649082>
>